lib/content-services/permission-manager/components/permission-display/permission-display.component.scss

```diff
@@ -0,0 +1,15 @@
+@mixin adf-permission-display-theme($theme) {
+
+    .adf{
+
+        &-display-permission-container {
+            display: flex;
+            justify-content: space-around;
+        }
+
+        &-datatable-permission {
+            display: flex;
+            flex-basis: 38%;
```

Comment:
not sure it is right flex-basis, how this thing behave in a small screen?

Reply:
this is just to say to flex : this is the base, we could improve adding the proper flex attribute and chose a behaviour for streched and not but keep in mind that we are basing this on datatable that is not working properly with small screens